otter_welcome_buddy/common/constants.py

```diff
@@ -5,6 +5,8 @@ class CronExpressions(Enum):
     """Defined Cron Expressions"""
 
     DAY_ONE_OF_EACH_MONTH_CRON: str = "0 0 1 * *"
+    # ENGLISH_CLUB_TIME: str = "2 20 * * 4"
+    ENGLISH_CLUB_TIME: str = "* * * * *"
```

Comment:
Please use line above as example, instead of ENGLISH_CLUB_TIME describe the behavior of the Cron. e.g. Run_ALL_DAYS_ALL_Hours